_templates/troubleshooting.md

```diff
@@ -0,0 +1,113 @@
+* Builds their confidence that the solution is proven and understood. 
+
+If the root cause is not known use the standard statement:  
+<!-- TO DO: We need to come up with proper statement, or use this? -->
+"The exact cause is not known at this time."
+
+<!-- TO DO: Add examples as at some stage. E.g.:
+        The exact cause is not known at this time.
+        This is expected behavior. <add concise description of why it is expected>
+        The schema used by the ODBC DataServer for the SQL database no longer matches the actual schema of the SQL database. -->
+
+
+## Solution
+<!-- Mandatory -->
+A Solution tells the reader what to do and how to do it 
+
+<!-- TO DO: We need to add sample statements for the different cases. E.g.:
+Use bullets or numbered steps and the commands needed to resolve the issue.
+
+-Article references an enhancement:
+
+	* State the enhancement number (if applicable) and, if implemented, the version where it is implemented.
+	* Use resolution template identified below depending upon the scenario:
+-For enhancement cases where there isn’t or we don’t know if there is an Idea:
+ 
+	"It was determined that the Progress Product is functioning as designed.
+ 
+	An enhancement to the product can be requested through the Progress Community via an Ideas submission.  Customer feedback is valuable and Idea submissions are monitored by our Product Management team.  Enhancement requests are reviewed during the planning phase of each new product release and a list of the enhancements chosen for implementation can be found in the Release Notes documents that accompany each release.  Once an Idea is submitted the Progress Software Community will have the opportunity to comment on and vote for the Idea.
+ 
+	For detailed information on how to submit an Idea, please refer to Knowledge Base article 000010839, <Smart Link to article 000010839>."
+ 
+-For enhancement cases where we know there is an Idea:
+
+	It was determined that the Progress Product is functioning as designed.
+ 
+	An enhancement request has been submitted as an Idea on the Progress Community.  To promote the Idea, click on this link: <link to Idea>.  Customer feedback is valuable and Idea submissions are monitored by our Product Management team.  Enhancement requests are reviewed during the planning phase of each new product release and a list of the enhancements chosen for implementation can be found in the Release Notes documents that accompany each release.  Once an Idea is submitted the Progress Software Community will have the opportunity to comment on and vote for the Idea.
+ 
+	For detailed information on how to submit an Idea, please refer to Knowledge Base article 000010839, <Smart Link to article 000010839>.
+
+-Article references a defect: 
+
+	* State the version or hot fix number where it is fixed. -->
+
+## Workaround
+<!-- Optional -->
+State the workaround to the issue. If multiple steps are needed, list them using bullets or numbered lists.
```

Comment:
State the workaround for the issue, if no solution is available. Should always begin with an introductory sentence which states that there is no solution. We should have template phrasing for it.